File: year2020/month11/day1113/ms_1113.cs
using System;

namespace algorithm_100
{
    class ms_1113
    {
        static void Main(string[] args)
        {
            String input = Console.ReadLine();

            if(input.Length <= 20)
            {
                for(int i = 0; i < input.Length; i++) 
                {
                    char temp = Convert.ToChar(input[i]);

                    if(!(('0' <= temp && temp <= '9') || temp == '.'))
                        input = input.Replace(temp, '0');
                }

                double data = Convert.ToDouble(input);

                Console.WriteLine(2* Convert.ToInt32(data));
                Console.WriteLine($"{data:0.00}");
            }
        }
    }
}

File: year2020/month11/day1113/ms_1113.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string input;

	getline(cin, input);

	if (input.length() <= 20) {
		for (int i = 0; i < input.length() - 1; i++) {
			char temp = input[i];

			if (!(('0' <= temp && temp <= '9') || temp == '.'))
				input.replace(i, 1, 1, '0');
		}

		cout << 2 * stoi(input) << endl;

		cout << fixed;
		cout.precision(2);
		cout << stod(input) << endl;
	}

	return 0;
}